packages/blockchain-utils-linking/src/__tests__/tezos.test.ts

```diff
@@ -84,4 +87,12 @@ describe('Blockchain: Tezos', () => {
       expect(result2).toMatchSnapshot()
     })
   })
+
+  describe('char2Bytes', () => {
+    test('should match the @taquito/utils char2Bytes() implementation', () => {
+      // get a random string of length 32
+      const s = 'some random string. this should be fine.'
+      expect(char2Bytes(s)).toBe(utils.char2Bytes(s))
+    })
+  })
```

Comment:
Not sure why you would add this test here. It's a really straight forward conversion.

Reply:
If `@taquito/utils` ever changes their implementation, it would be nice to know rather than having a weird bug that's hard to debug.

Reply:
@Geo25rey It is not `@taquito/utils` that is important, it is `TezosProvider.signer.sign` that one should care about. As long as the latter accepts hex, `@taquito/utils` can be removed. And, if by some chance `TezosProvider.signer.sign` stops accepting hex, the tests will warn us. So, please, remove it from the dependencies list. It brings nothing useful here.

Reply:
That makes sense to me. I am on it.